beacon_node/network/src/beacon_processor/work_reprocessing_queue.rs

```diff
@@ -770,6 +845,32 @@ impl<T: BeaconChainTypes> ReprocessQueue<T> {
                     }
                 }
             }
+            InboundEvent::ReadyBackfillSync(queued_backfill_batch) => {
+                let seconds_from_slot_start = slot_clock
+                    .seconds_from_current_slot_start()
+                    .map_or("unknown".to_string(), |duration| {
+                        duration.as_secs().to_string()
+                    });
+
+                debug!(
+                    log,
+                    "Sending scheduled backfill work to Beacon Processor at {} seconds from current slot start",
+                    seconds_from_slot_start
+                );
```

Comment:
```suggestion
                debug!(
                    log,
                    "Sending scheduled backfill work";
                    "seconds_from_slot_start" => seconds_from_slot_start
                );
```

A suggestion to make the log more consistent with out other logs. We try to separate the log into a short description and then key-value pairs of additional info.

Keeping the description short keeps the log lines looking neat with some space before the key-value pairs start. Using key-value pairs helps logging systems parse the data (especially when we emit the logs as structured JSON).

Reply:
I like this 👍